### PR TITLE
add copyright notice to figcaption

### PR DESCRIPTION
**Pull request by andygnewman**
Put the image copyright notice into the figcaption, appended to longdesc when that is present.
Swap from deprecated font styling to oTypography

### CODE REVIEW

File: test/server/stylesheets/external-image.test.js [hunk 1]
```diff
@@ -321,6 +321,54 @@ describe('External images', () => {
...
+			return transform(
+				'<html>' +
+					'<body>' +
+						'<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>' +
+						'<p><img src="http://my-image/image.jpg" longdesc="This is a long description" data-copyright="© Bloomberg" width="800" height="600" /></p>' +
+						'<p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>' +
+					'</body>' +
+				'</html>\n'
+			)
+			.then((transformedXml) => {
```
Comment (matthew-andrews) [inline test/server/stylesheets/external-image.test.js]: It really makes no difference at all but did you know that the brackets are optional in fat arrow functions when you only have one variable?
`.then(variable => {`

**Comment by matthew-andrews** [comment]
:+1: :+1: (one for the change and one for the baby step toward origamification)

File: test/server/stylesheets/external-image.test.js (hunk 1 above)
Comment (matthew-andrews) [inline test/server/stylesheets/external-image.test.js]: I can understand if you would want to keep the brackets for single variable fat arrows for consistency though.

Comment (andygnewman) [inline test/server/stylesheets/external-image.test.js]: Yes I did know that, but was keeping consistency as you said.

Comment (matthew-andrews) [inline test/server/stylesheets/external-image.test.js]: :+1: quite a nice idea. Keeps diffs cleaner. I might start doing the same.

**Pull request merged**